assets/src/styles/blocks/OldCovers/TakeActionCovers.scss

```diff
@@ -223,56 +223,56 @@
       }
     }
   }
-}
-
-.cover-card-tag {
-  --block-covers--card-tag-- {
-    color: $yellow;
 
-    &:hover {
-      text-decoration: underline;
+  .cover-card-tag {
```

Comment:
I've had to move these selectors inside the `.cover-card` one, otherwise they were still applied to the new styles even though they shouldn't